module/modmanager/manager.go

```diff
@@ -67,15 +70,15 @@ type module struct {
 	// pendingRemoval allows delaying module close until after resources within it are closed
 	pendingRemoval bool
 
-	// inRecovery stores whether or not an OnUnexpectedExit function is trying
-	// to recover a crash of this module; inRecoveryLock guards the execution of
+	// inStartup stores whether or not an OnUnexpectedExit function is trying
+	// to recover a crash of this module; inStartupLock guards the execution of
 	// an OnUnexpectedExit function for this module.
```

Comment:
```suggestion
	// inStartup stores whether or not the manager or the OnUnexpectedExit function is trying
	// to startup this module; inStartupLock guards the execution of
	// an OnUnexpectedExit function for this module.
```
kind of?

Reply:
Pushed this change (wanted `manager of` not `manager or`)!